lib/matplotlib/legend.py

```diff
@@ -208,8 +209,11 @@ def _update_bbox_to_anchor(self, loc_in_canvas):
     Whether round edges should be enabled around the `~.FancyBboxPatch` which
     makes up the legend's background.
 
-shadow : bool, default: :rc:`legend.shadow`
+shadow : bool or color, default: :rc:`legend.shadow`
     Whether to draw a shadow behind the legend.
+    If value is a color, a shadow of that color will be applied.
+    If the value is neither boolean nor a valid color, behavior is that of
```

Comment:
@timhoffm Do we use the concept of "truthy" in the docs?

Reply:
Good question. I only found one instance (axes3d.py).

Reply:
In general, we use "truthy" simplifications, and here we have done so before (`if shadow:`). IMO that's ok with pure bool parameters.

However, now we also accept color-like expressions, which can subtly be valid or not:
~~~
is_color_like('r')  --> True
is_color_like('s')  --> False
is_color_like('lightblue') --> True
is_color_like('lightred')  --> False
is_color_like('#ff0000') --> True
is_color_like('ff0000')  --> False
is_color_like('0.5') --> True
is_color_like(0.5)   --> False
~~~

Therefore, I recommend to explicitly enforce the type or error out:
~~~
if is_color_like(shadow):
    ...
elif shadow is True:
    ...
elif shadow is False:
    pass
else:
    raise ValueError('shadow must be a valid color or bool')
~~~

Otherwise, invalid color specifications may pass without knowing.

Reply:
Did this exact thing.